oaknational/Oak-Web-Application
# feat: update mv versions for downloadable resources [LESQ-668]

## Description

Music year: 1971

_in draft until MVs are released to production_

- update mv versions for lesson overview and downloads

## Issue(s)

Fixes #
 missing downloadable resources button on page with downloads

## How to test

1. Go to https://deploy-preview-2286--oak-web-application.netlify.thenational.academy/teachers/programmes/science-primary-ks2/units/earth-sun-and-moon/lessons/the-shape-of-earth
2. Click on download all
3. You should see resources for download
